lib/Parser/Parse.h

```diff
@@ -472,29 +472,23 @@ class Parser
     bool NextTokenIsPropertyNameStart() const { return m_token.tk == tkID || m_token.tk == tkStrCon || m_token.tk == tkIntCon || m_token.tk == tkFltCon || m_token.tk == tkLBrack || m_token.IsReservedWord(); }
 
     template<bool buildAST>
-    void PushStmt(StmtNest *pStmt, ParseNodePtr pnode, OpCode op, ParseNodePtr pnodeLab, LabelId* pLabelIdList)
+    void PushStmt(StmtNest *pStmt, ParseNodePtr pnode, OpCode op, LabelId* pLabelIdList)
     {
-        AssertMem(pStmt);
```

Comment:
Asserting a pointer is not null before using it is redundant, since it will AV anyway.  You could say perhaps we gain benefit by asserting earlier, but I have never seen any benefit to such asserts in practice.

I actually have another change pending that removes all the legacy assert macro usage.